source/ed/readme.md

```diff
@@ -25,6 +26,24 @@ example: ```--connection-string="host=localhost user=my_user password=my_pwd dbn
 
 Note: importing data will remove the data of the specified type (i.e. osm2ed and geopal2ed will remove street network data).
 
+## Setup `ed` database
+
+```bash
+sudo -i -u postgres
```

Comment:
You might want to create a user if it's missing. Otherwise the db creation will fail.

```shell
sudo -u postgres createuser my_user
sudo -u postgres createdb -O my_user my_db
sudo -u postgres psql -d my_db -c "CREATE EXTENSION postgis;"
```

Reply:
Yep and it has to have creation right, I hesitated to put the following (but thought it's more postgresql doc)

```bash
CREATE USER my_user;
ALTER ROLE my_user WITH CREATEDB;
ALTER USER my_user WITH ENCRYPTED PASSWORD 'my_pwd';
```

I'm OK to add something like that if you guys think it's worthy.